server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/dto/AddRegexMappingRequestBodyDTO.java

```diff
@@ -0,0 +1,25 @@
+package org.apache.james.webadmin.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AddRegexMappingRequestBodyDTO {
```

Comment:
I think `RegexMappingDTO` would be enough for a class name here

Reply:
Okey, I have refactored the class name.